config/zookeeper_persister_test.go

```diff
@@ -26,24 +27,24 @@ func TestMain(m *testing.M) {
 	servers = make([]string, 1)
 	servers[0] = fmt.Sprintf("localhost:%d", t.Servers[0].Port)
 
-	createExistingNode("/existing")
-	createExistingNode("/historic")
-	r := m.Run()
-
-	os.Exit(r)
+	os.Exit(m.Run())
 }
 
 func TestNewPathError(t *testing.T) {
-	p, err := NewZkConfigPersister("/LOCAL/quotaservice", servers)
+	conn := setup()
+
+	p, err := NewZkConfigPersisterWithConnection("/LOCAL/quotaservice", conn)
 
 	if err == nil {
+		defer p.(*ZkConfigPersister).Close()
 		t.Error("Should have received error on new because path does not exit")
-		p.(*ZkConfigPersister).Close()
 	}
 }
 
 func TestNew(t *testing.T) {
-	p, err := NewZkConfigPersister("/quotaservice", servers)
+	conn := setup()
+
+	p, err := NewZkConfigPersisterWithConnection("/quotaservice", conn)
```

Comment:
Maybe make these `/existing` and `/quotaservice` strings constants? I see them in a few places.